### PR TITLE
Update .gitignore to ignore dsp parts

### PR DESCRIPTION
**Pull request by AlwinEsch**
This request add the ignored fields related to audio DSP. Think this can be added separate from from the planned #4402 to prevent also git add problems during branch chance.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -926,3 +932,4 @@ lib/cpluff/stamp-h1
 /addons/library.xbmc.pvr/libXBMC_pvr.lib
 
 /pvr-addons
+/adsp-addons
```
Comment (garbear) [inline .gitignore]: this isn't needed. `/pvr-addons` was for the old xbmc-pvr-addons repo, where all the add-ons were combined in a single repo. The build system knew to look in the `/pvr-addons` folder for these add-ons.

Comment (AlwinEsch) [inline .gitignore]: I remove the /adsp-addons there, has forgot this.

There is still for /pvr-addons a:
ifneq ($(wildcard pvr-addons/Makefile.am),)
BOOTSTRAP_SUBDIRS += pvr-addons/configure.ac
endif
inside bootstrap.mk, also remove this?

**Comment by AlwinEsch** [comment]
From my side OK then to bring in. Allowed to merge direct, jenkin make no sense for me there.

**Comment by Montellese** [comment]
Why are we already adding this and not as part of #4402 ? Adding stuff to gitignore that doesn't exist IMO doesn't make much sense.

**Comment by AlwinEsch** [comment]
DSP was already planned for next version and thought to bring it in separate to prevent also problems during update of branch where I change to master and back and to reduce amount of changed files.
Can leave it if you wish inside #4402.

**Pull request closed without merging**